hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/ServerRemoteProcedure.java

```diff
@@ -137,6 +140,10 @@ synchronized void remoteOperationDone(MasterProcedureEnv env, Throwable error) {
         getProcId());
       return;
     }
+    //below persistence is added so that if report goes to last active master, it throws exception
+    state = MasterProcedureProtos.ServerRemoteProcedureState.SERVER_REMOTE_PROCEDURE_REPORT_SUCCEED;
```

Comment:
If there is an error, we should not marked it as succeed? And we also need to persist the error field?

Reply:
I have changed the status. For persisting the error field I am thinking about it. I did see an example of persisting the exception. If need to persist I think I can use those.